<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgShowCapabilities</class>
 <widget class="QDialog" name="dlgShowCapabilities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètres de l'application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	border: 1px solid #ffffff;
	border-radius: 1px;
	background: #818181;
	padding: 0px 3px 0px 3px;
	color: #ffffff;
}

QTableView QHeaderView::section{
		border: 1px solid #ffffff;
		background-color: #818181;
		color: #ffffff;
    }
	</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="butLay">
     <item>
      <widget class="QPushButton" name="buttResize">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/btn_resize</normaloff>:/resources/btn_resize</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttHsp">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttValid">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="capacitiesTwg">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="capkeyTab">
      <attribute name="title">
       <string>Paramètres généraux</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="extLab">
         <property name="text">
          <string>Possibilité d'extraire le RFU</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="majLab">
         <property name="text">
          <string>Possibilité de mettre à jour le RFU</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="extLed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lngLab">
         <property name="text">
          <string>Longueur maximale autorisée du rectangle d'extraction</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="cmtLab">
         <property name="text">
          <string>Commentaire</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="capkeyVsp">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lngLed">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cmtLed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="majLed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="caputilTab">
      <attribute name="title">
       <string>Paramètres de l'utilisateur</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableView" name="caputilTbv">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
